Assets/Plugins/Editor/Uplift/UpdateTool/Updater.cs

```diff
@@ -0,0 +1,131 @@
+using UnityEngine;
+using UnityEditor;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using Uplift.Common;
+using Uplift.Windows;
+using System.Net;
+using System.Linq;
+using System;
+using System.Globalization;
+
+namespace Uplift.Updating
+{
+    public class Updater : MonoBehaviour
+    {
+        private static readonly string dateFormat = @"MM\/dd\/yyyy HH:mm";
+        private static readonly CultureInfo provider = CultureInfo.InvariantCulture;
+        private static readonly string lastUpdateCheckKey = "UpliftLastUpdateCheck";
+        private static IEnumerator updateCoroutine;
+
+        public static void UpdateUplift(string url)
+        {
+            string unitypackageName = url.Split('/').Last();
+            DirectoryInfo assetsPath = new DirectoryInfo(Application.dataPath);
+            string destination = Path.Combine(assetsPath.Parent.FullName, unitypackageName);
+            
+            ServicePointManager.ServerCertificateValidationCallback = delegate { return true; };
```

Comment:
We should be careful here. I am not fond of bypassing SSL.

See http://www.khalidabuhakmeh.com/validate-ssl-certificate-with-servicepointmanager for I think a better solution.